File: Java/JavaProject/src/猜数字.java
import java.util.Scanner;
import java.util.Random;
public class 猜数字 {
    public static void main(String[] args){
        Scanner input = new Scanner(System.in);
        Random rd = new Random();
        int number = rd.nextInt(100)+1;
        boolean result = true;
        System.out.print("猜数字游戏开始，请输入数字：");
        while(result)
        {
            int inputNumber=input.nextInt();
            if(inputNumber==number){
                System.out.println("恭喜你，猜对了");
                result = false;
            }
            else if(inputNumber>number)
                System.out.print("你输入的数字比随机数大，请再猜：");
            else if(inputNumber<number)
                System.out.print("你输入的数字比随机数小，请再猜：");

        }
    }
}

File: Python/Python基础学习之路/猜数字.py
import random

print("猜数字游戏开始，请输入数字：")
count = 0
number: float = random.randint(0, 10)
print(number)
while count < 3:
    inputNumber = input()
    count += 1
    if count < 3:
        if number == int(inputNumber):
            print("恭喜你，猜对了!游戏结束！")
            count = 3
        elif number > int(inputNumber):
            print("你输入的数字比随机数大，请再猜：")
        elif number < int(inputNumber):
            print("你输入的数字比随机数小，请再猜：")
    else:
        print("你的三次机会用完了！请重新开始游戏！")
